docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -168,8 +171,34 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 <1> `HotBackup` resource name used for restore. The backup folder name will be taken from the `HotBackup` resource.
 <2> Agent responsible for restoring data from the local storage.
-    The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
-    uses the latest agent version that is compatible with its version.
+The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
+uses the latest agent version that is compatible with its version.
+
+To restore from local backups that were not created by the HotBackup resource, configure `localConfig`.
+
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hazelcast-persistence-restore-local.yaml[]
+----
+
+<1> The name of the cluster for both backup and restore must be the same. If the cluster name is different, the restore fails.
+<2> If you configured maps with persistence, you need to create them while creating Hazelcast CR  as well. To be able to create them while creating Hazelcast CR, you need to use xref:custom-config.adoc[Custom Config]. If you try to apply Hazelcast CR and then Map CRs, it will not work.
+<3> `pvcNamePrefix` is the prefix of the existing PVCs. It can be `persistence` or `hot-restart-persistence` depending on the installation method (in different versions of Hazelcast Helm Chart and Hazelcast Platform Operators these two values are used by default) of the old cluster. You can decide this value by running `kubect get pvc` command.
```

Comment:
```suggestion
<3> `pvcNamePrefix` is the prefix of the existing PVCs. It can be set to `persistence` or `hot-restart-persistence` depending on the installation method of the old cluster. In different versions of Hazelcast Helm Chart and Hazelcast Platform Operator, these values are used by default. Run the `kubect get pvc` command to decide which setting best suits your requirements.
```